demos/embeddings/watch-your-step-cora-demo.ipynb

```diff
@@ -0,0 +1,449 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from stellargraph.core import StellarGraph\n",
+    "from stellargraph.mapper import AdjacencyPowerGenerator\n",
+    "from stellargraph.layer.watch_your_step import WatchYourStep\n",
+    "from stellargraph.utils.loss import graph_log_likelihood\n",
+    "\n",
+    "from matplotlib import pyplot as plt\n",
+    "from tensorflow.keras import optimizers, Model, layers, regularizers\n",
+    "from sklearn import preprocessing, feature_extraction, model_selection\n",
+    "\n",
+    "import networkx as nx\n",
+    "import random\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Loading in the data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_dir = os.path.expanduser(\"~/data/cora\")\n",
```

Comment:
This can now use the new `datasets` module, e.g.:

https://github.com/stellargraph/stellargraph/blob/205b6d81ccedb7be20c0d1cd8843bb4cf819a409/demos/node-classification/gcn/gcn-cora-node-classification-example.ipynb#L63-L65

Reply:
didn't realise we had this functionality, nice!

Reply:
New in #690.